preview/scripts/edit-description.js

```diff
@@ -1,7 +1,7 @@
 module.exports = async ({inputs, github, context}) => {
     const PR_NUMBER = context.issue.number;
     const RTD_PROJECT_SLUG = inputs["project-slug"];
-    const RTD_PROJECT_LANGUAGE = inputs["project-language"];
+    const RTD_PROJECT_LANGUAGES = inputs["project-languages"].split(",");
```

Comment:
Instead of splitting it by `,`, this attribute could be a list, right? Like

```yaml
project-languages:
  - en
  - ja
```

Reply:
<img width="572" alt="image" src="https://user-images.githubusercontent.com/31057849/223622031-b83bf85a-8bef-40ac-a39b-fb96550c51e0.png">
It seems that only string, number, and boolean types are possible as arguments.